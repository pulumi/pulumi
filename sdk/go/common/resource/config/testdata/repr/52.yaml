my:name:
- a
- value
- c
