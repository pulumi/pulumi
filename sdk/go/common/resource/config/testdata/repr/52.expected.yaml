decrypt:
  my:name: '["a","value","c"]'
paths:
  my:name:
    value:
    - a
    - value
    - c
    string: '["a","value","c"]'
    redacted: '["a","value","c"]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAALf+EKgADBnN0cmluZwwDAAFhBnN0cmluZwwHAAV2YWx1ZQZzdHJpbmcMAwABYw==
    secure: false
    isObject: true
  my:name[0]:
    value: a
    string: a
    redacted: a
    object: DhAABnN0cmluZwwDAAFh
    secure: false
    isObject: false
  my:name[1]:
    value: value
    string: value
    redacted: value
    object: EhAABnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: false
  my:name[2]:
    value: c
    string: c
    redacted: c
    object: DhAABnN0cmluZwwDAAFj
    secure: false
    isObject: false
